### ISSUE TITLE
LR 4000E7E4 - hekate exception occurred

### ISSUE THREAD

**Issue by innocentDE** [CTCaer/hekate#1047]
Mod Chip installation with Picofly RP2040. Hang detected in LP0/Minerva.

First time this happened. Freshly installed. Modded already a few.

**Comment by innocentDE** [comment]
- blue Screen when mod chip is removed
- boots into hekata (nyx is missing) when removing the SD Card
- emmc and sd card are find (sd card is legit and properly formatted)
- tools and console info from hekate are working

**Comment by urherenow** [comment]
Did you search the closed issues for this error? It's almost always related to sd card/reader problems, file corruption, or you need to reflow your ram or apu. With an RPI, It could also be a wire that's a tad too long, or 1 of 2 spots that may need an extra 45ohm resistor (silicon lottery with that issue, I think).

**Comment by pmnlla** [comment]
Blue screen is usually a bad NAND image or you need to reflow your board, iirc.

**Comment by innocentDE** [comment]
> Blue screen is usually a bad OS image or you need to reflow your board, iirc.

Alright, will try to reflow. Thanks

**Comment by innocentDE** [comment]
No, neither did i find a solution nor did I fix it.

**Comment by leofrachet11** [comment]
I just found something really important and I was wondering if it works on your switch as well.

I'm was still trying to find why when there is no SD card installed, everything works fine, so I didn't install the SD card and it said "No SD Card" as I expected, but I found that when booting, if you hold both the volume down and volume up button at the same time, it bypasses this screen and boots into the OFW, and it worked! There were no issues or blue screens, proving that the issue isn't with the switch, so I was wondering if this would work on your one as well.

**Comment by CTCaer** [comment]
That's probably a hw issue or your chip corrupts hekate when it's loaded.
Probably a reflow of the SoC will fix it.

**Comment by KlausDEngel** [comment]
Delete or rename bootloader/sys/libsys_minerva.bso
Hekate will warn that Minerva is missing and run RAM untrained (slightly slower), but it should boot and let you launch CFW. This isolates the crash to the Minerva module.

**Comment by leofrachet11** [comment]
For me this was a RAM chip solder issue, if I manually pushed down hard on one of the chips it would boot and everything would work normally, so I reflowed that chip and it worked, so I’m not sure that anything to do with the software would fix anything (especially the fact that when the modchip was removed there was a blue screen)

**Comment by KlausDEngel** [comment]
Yes, definitely an hardware issue with the RAM. Some devices work in handheld mode but crash in docked mode. Renaming libsys_minerva.bso solves the crash problem for hekate, at least for handheld mode.
